application/src/main/java/org/thingsboard/server/service/edge/rpc/EdgeGrpcService.java

```diff
@@ -253,6 +274,8 @@ public void deleteEdge(TenantId tenantId, EdgeId edgeId) {
         EdgeGrpcSession session = sessions.get(edgeId);
         if (session != null && session.isConnected()) {
             log.info("[{}] Closing and removing session for edge [{}]", tenantId, edgeId);
+            session.destroy();
+            session.deleteTopic(edgeId);
```

Comment:
I think deleteTopic shouldn't be in the common interface, as it related only to Kafka